.github/workflows/ci.yml

```diff
@@ -64,6 +64,9 @@ jobs:
       - name: Generate Files
         run: earthly --strict --remote-cache ghcr.io/crossplane/earthly-cache:${{ github.job }} +generate
 
+      - name: Tidy Modules
```

Comment:
the `generate` target used to do the tidy before the generate - should we maintain that order here in the CI check?

Reply:
I can't think of a reason the order would matter. Can you?